docs/product-user-guide/src/main/asciidoc/interacting-with-processes-migrating-process-designer-proc.adoc

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
I'm not sure how this topics relates to this document.
This has nothing to do with interacting with processes and tasks as a knowledge worker.

Reply:
Yes, I am also aware that this topic is not related to current topic at hand. I have just documented the migration procedure and have finalized a plan to move this topic to some other document.

Change ack.